.github/workflows/golangci-lint.yaml

```
@@ -0,0 +1,20 @@
+---
+name: "golangci-lint"
```

Please change:
```suggestion
name: "Golang Lint"
```
I've been trying to keep unified naming in CIs lately.